[nmstate-1.0] ovs: remove ovs-port prefix

cathay4t opened the pull request:
Prepending the prefix causes ovnkube-node pods to crash.
This change adds -port postfix to ovs-interface connection instead.

ffmancera reviewed (changes requested):
Hi Gris, is this breaking something? I think this is failure is new: https://github.com/nmstate/nmstate/runs/4262972477?check_suite_focus=true#step:6:2049

Commit:
ovs: remove ovs-port prefix

Prepending the prefix causes ovnkube-node pods to crash.
This change adds -port postfix to ovs-interface connection instead.

Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

cathay4t commented:
> Hi Gris, is this breaking something? I think this is failure is new: https://github.com/nmstate/nmstate/runs/4262972477?check_suite_focus=true#step:6:2049

Fixed.

ffmancera reviewed (approved):
LGTM! Thanks!
